# estic.doc
# Mir ist mitgeteilt worden, daß die FreeBSD Version das Flag USE_OLD_TTY
# braucht. Offenbar ist die Behandlung der RS232 Handshakeleitungen
# geändert worden, die für die neue Platine der Istec (#3) benötigt
# werden. Nähere Informationen liegen mir nicht vor, ich wäre dankbar,
# wenn das nochmal jemand verifizieren könnte.

# ovb@swissmail.com does not use -DUSE_OLD_TTY
# neither does jhs@freebsd.org need it

*** estic/make/freebsd.mak.orig	Sat Sep 14 15:40:16 1996
--- estic/make/freebsd.mak	Mon Nov 18 03:08:31 1996
***************
*** 35,41 ****
  # Both configurations of CFLAGS will probably work since FreeBSD has a
  # smart linker...
  #CFLAGS = -DFREEBSD -DUSE_OLD_TTY -g -Wall -I$(INCDIR) -x c++
! CFLAGS	= -DFREEBSD -DUSE_OLD_TTY -g -Wall -I$(INCDIR) -x c++ -fno-implicit-templates -DEXPLICIT_TEMPLATES
  
  
  # ------------------------------------------------------------------------------
--- 35,42 ----
  # Both configurations of CFLAGS will probably work since FreeBSD has a
  # smart linker...
  #CFLAGS = -DFREEBSD -DUSE_OLD_TTY -g -Wall -I$(INCDIR) -x c++
! #CFLAGS	= -DFREEBSD -DUSE_OLD_TTY -g -Wall -I$(INCDIR) -x c++ -fno-implicit-templates -DEXPLICIT_TEMPLATES
! CFLAGS	= -DFREEBSD -g -Wall -I$(INCDIR) -x c++ -fno-implicit-templates -DEXPLICIT_TEMPLATES
  
  
  # ------------------------------------------------------------------------------
